external-crates/move/crates/move-disassembler/src/disassembler.rs

```diff
@@ -445,16 +576,67 @@ impl<'a> Disassembler<'a> {
                     },
                 )?;
 
-                any_write!(buffer, " }}")
+                any_write!(buffer, " }}")?;
+                // perhaps surprisingly, but the location is of the whole variant
+                // and not just of its name
```

Comment:
This is indeed surprising. Maybe we should make it just be of its name?

Reply:
Didn't want to sneak it in here, but can certainly do it in a separate PR if need be